doc/classes/CollisionShape3D.xml

```diff
@@ -29,6 +29,13 @@
 		</method>
 	</methods>
 	<members>
+		<member name="debug_color" type="Color" setter="set_debug_color" getter="get_debug_color" default="Color(0, 0, 0, 0)">
+			The collision shape debug color.
+			[b]Note:[/b] The default value is [member EditorSettings.editors/3d_gizmos/gizmo_colors/shape]. The [code]Color(0, 0, 0, 0)[/code] value documented here is a placeholder, and not the actual default debug color.
```

Comment:
The note is misleading as if to hide how this actually works. What it entails is probably something more like this:
```suggestion
			The collision shape's debug color. If this value is set to [code]Color(0, 0, 0, 0)[/code], the value of [member EditorSettings.editors/3d_gizmos/gizmo_colors/shape] is used, instead.
```

Reply:
It was originally written that way to keep it consistent with CollisionShape2D. Your suggestion reads a lot more like a proper description though, so I've rewritten it similarly.